plugins/bazaar-backend/src/service/router.ts

```diff
@@ -83,8 +83,14 @@ export async function createRouter(
     response.json({ status: 'ok', data: data });
   });
 
-  router.get('/projects', async (_, response) => {
-    const data = await dbHandler.getProjects();
+  router.get('/projects/:limit?/:order?', async (request, response) => {
```

Comment:
Hm is this the right syntax? I thought maybe just `/projects`, and then on the line below do e.g. `const limit = request.query.limit?.toString()` and then the limit will be a string or an undefined. You'd then call the endpoint as `curl .../projects?limit=10`